demisto_sdk/commands/common/constants.py

```diff
@@ -820,6 +821,7 @@ def found_hidden_param(parameter_name):
     MISC_DIR,
     CONNECTIONS_DIR,
     INDICATOR_FIELDS_DIR,
+    TESTS_DIR
```

Comment:
Explain this change.

Reply:
Since base validator sometimes need to open conf.json, and conf.json is in 'Tests' folder, Added **Tests/cong.json** in the setup of test validator's test and removed it in the tearDown.

Since **Tests** folder was not created needed to import it in order to create it, makes sense?